src/components/Breadcrumb/BreadcrumbList.tsx

```diff
@@ -30,22 +30,24 @@ function BreadcrumbList({
     ...otherProps
 }: IBreadcrumbListProps) {
     const renderBreadcrumb = (propsBreadcrumb) => {
+        const {onClick, ...otherProps} = propsBreadcrumb;
         return (
             <BreadcrumbItem
-                {...propsBreadcrumb}
-                /*itemDivider="/"*/ onClick={
+                /*itemDivider="/"*/
+                {...otherProps}
+                onClick={
                     onItemClick
                         ? (e) => {
                               onItemClick(propsBreadcrumb.href, e);
                           }
-                        : undefined
+                        : onClick
```

Comment:
What is the use case of `onItemClick` being undefined?
Assigning `onClick` for all breadcrumb items (except the last) instead does not seem to make a lot of sense.